_projects/ak-medicaid.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: project
+title: Helping Those in Need
+description: Using Agile methodologies and DevOps to overhaul the State of Alaska's Medicaid system.
+permalink: /projects/ak-medicaid/
+tags: project
+image: /assets/img/project-images/alaska.jpg
```

Comment:
Add caption? People in a field doesn't obviously connect to modernizing a technology system.

Reply:
There isn't really a connection. I had a hard time finding a picture that was related that was available under a permissive license. Good call on captions though, I'll spin that off into a separate issue.